camera_matrix:
- [303.32388836352, 0.0, 686.9139968591608]
- [0.0, 301.5277891953513, 521.3586002402685]
- [0.0, 0.0, 1.0]
distortion_coefficients: [0.019771511947000206, -0.014553826443000843, 0.008699032851348088, -0.0022248207705199276]
size:
    rows: 1384
    cols: 1032
